src/features/json/projectJSONContribution.ts

```diff
@@ -0,0 +1,236 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+'use strict';
+
+import {MarkedString, CompletionItemKind, CompletionItem, DocumentSelector} from 'vscode';
+import {XHRRequest, XHRResponse, getErrorStatusDescription} from 'request-light';
+import {IJSONContribution, ISuggestionsCollector} from './jsonContributions';
+import {Location} from 'jsonc-parser';
+
+import * as nls from 'vscode-nls';
+const localize = nls.loadMessageBundle();
+
+const FEED_INDEX_URL = 'https://api.nuget.org/v3/index.json';
```

Comment:
Yes, that would be a future improvement. The feeds to use should probably be picked up from nuget.config